src/PortfolioPlanning/Components/Plan/AddItemPanel.scss

```diff
@@ -64,12 +62,23 @@
     margin-top: 5px;
 }
 
+.workItemTypeSectionBody {
+    margin-left: 10px;
```

Comment:
In general we should try and use the defined styles by azure devops. They're on an 8px scale and you can use $spacing-8, $spacing-16 for consistency.

Reply:
Fixed... waiting for Pranoti's revert to merge this one.